.eslintrc.js

```diff
@@ -135,7 +135,7 @@ module.exports = {
     'react/no-string-refs': 1,
     'react/button-has-type': 1,
     'react/jsx-props-no-spreading': 1,
-    'react/jsx-one-expression-per-line': 1,
+    'react/jsx-one-expression-per-line': 0,
```

Comment:
Is this supposed to be in this PR?

Reply:
Yes, see https://dsva.slack.com/archives/CBU0KDSB1/p1643665837374739?thread_ts=1643662645.374679&cid=CBU0KDSB1 - this rule was making eslint "fix" the problem, but instead made an error.

Reply:
Gotcha, I was kinda wondering if some of those linting rules that were enabled were going to cause headaches. Looks like that is the case